src/recipe.js

```diff
@@ -7,12 +7,7 @@ class Recipe {
     this.ingredients = recipe.ingredients
     this.instructions = recipe.instructions
   }
-
-  calculateIngredientCost() { // need to create test for this
-    return this.ingredients.map(ingredient => {
-      return (ingredient.cost * ingredient.quantity.amount) / 100
-    })
-  }
+  
```

Comment:
I'm not entirely sure I wanna go ahead with this but I'm open to discussing it in our stand up!

Reply:
Yes, we should definitely discuss!